tslint.json

```diff
@@ -18,7 +18,7 @@
         "timeEnd",
         "trace"
     ],
-    "no-consecutive-blank-lines":  true,
+    "no-consecutive-blank-lines": [true],
```

Comment:
I don't disagree &ndash; but tslint nevertheless flags its own config file on account of this.